### PR TITLE
Azzam Alkhaldi

### PR DESCRIPTION
**Pull request by AlkhaldiAzzam**
Your Name:

Azzam Alkhaldi

How difficult did you find this (0 - 10)? (0 very easy, 10 very difficult)
4
How much of this did you complete? (0-10)
10

How long did it take?
20 minutes

Was there anything that you struggled with? If so, what?

Is there anything that you would like some further information on?
Do you have any suggestions to improve this assignment?

### CODE REVIEW

**Comment by AlkhaldiAzzam** [comment]
almost all of the questions ended up having an open curly brace instead of a closing one

File: homework.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+ const longestWord = function(str){
+
+let longStr = "";
+
+
+str = str.split("");
+
+for (let i = 0; i < str.length; i++){
+
+if (str[i].length > longWord)
```
Comment (alanoud-GA) [inline homework.js:177]: `longWord` was never defined, this will throw you an error.

File: homework.js [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+for (let i = 0; i <= n; i++){
+if (n % 2==0)
+sum +=i;
+
+
+}
+ 
+
+return sum;
+     {
```
Comment (alanoud-GA) [inline homework.js:38]: why are your closing curly brackets flipped?

File: homework.js [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+{
+
+
+
+
+
+ 10-
+
+ const largestEvenNum = function(arr){
+let largeNum = unidentified;
```
Comment (alanoud-GA) [inline homework.js:197]: what is `unidentified `?

**Pull request closed without merging**